### PR TITLE
Fix UA being unavailable during user account setup

### PR DESCRIPTION
**Pull request by HighCrit**
Also forgot one User Agent instance.

### CODE REVIEW

File: common/src/main/java/com/wynntils/core/webapi/WebManager.java [hunk 1]
```diff
@@ -76,19 +76,21 @@ public final class WebManager {
 
     private static List<MapProfile> maps = null;
 
-    private static String userAgent;
+    private static final String USER_AGENT;
```
Comment (magicus) [inline common/src/main/java/com/wynntils/core/webapi/WebManager.java]: Can't you do like: 
```
    private static final String USER_AGENT = String.format(...);
```
?

Comment (magicus) [reply common/src/main/java/com/wynntils/core/webapi/WebManager.java]: static clinit blocks are the worst...

Comment (HighCrit) [reply common/src/main/java/com/wynntils/core/webapi/WebManager.java]: During testing (on Artemis) yesterday VERSION was still an empty string and the static block fixed it. But am unable to reproduce it now 🤔.

Comment (magicus) [reply common/src/main/java/com/wynntils/core/webapi/WebManager.java]: We're kind of doing the version setup backwards, but the way I read the code the version string should be set at the second line in WynntilsMod.init(), and the WebManager not inited until later in that method. I don't know if we have some racy thing calling methods on the WebManager before that, causing it to be loaded too early.

Comment (magicus) [reply common/src/main/java/com/wynntils/core/webapi/WebManager.java]: In any case, the static block would not fix anything, so it was probably just pure luck, if there is a race.

**Review by magicus — APPROVED** [review]

**Pull request merged**